setup.py

```diff
@@ -18,7 +15,7 @@ def _compile_ccl(debug=False):
            "-DPYTHON_VERSION=%d.%d.%d" % (v.major, v.minor, v.micro)]
     if debug:
         cmd += ["-DCMAKE_BUILD_TYPE=Debug"]
-    if call(cmd) != 0:
+    if call(cmd, shell=True) != 0:
```

Comment:
Try without `shell=True` since it is added in python 3.12, but in GitHub Actions we use python 3.8.